spec/models/patron_spec.rb

```diff
@@ -3,5 +3,64 @@
 require 'rails_helper'
 
 RSpec.describe Patron, type: :model do
```

Comment:
```suggestion
RSpec.describe Patron do
```
because of `config.infer_spec_type_from_file_location!`

Reply:
Fixed in #32